### ISSUE TITLE
Feature: Want to add a Python Project  (Automatic Number Plate Recognition)

### ISSUE THREAD

**Issue by divyachopra14** [keshavsingh3197/hacktoberfest#4053]
### Is your feature request related to a problem? Please describe.
Yes, want to add a Automatic Number Plate Recognition project, which is a computer vision project using OpenCV and EasyOCR.

Want to add more good projects to add value to the repository 

### Describe the solution you'd like

I would like to submit a full fledged project that can detect car number plates and can extract the license number out of it into text. 

### Describe alternatives you've considered

This Project could also be done using tensorflow and keras object detection libraries for better accuracy.

### Additional context

Please assign this issue to me under the tag of hacktoberfest 2022

**Comment by ragnarok-02** [comment]
Can you please assign this to me?

**Comment by theshreyaa** [comment]
Hey ! I would love to work on this by making my first Hacktoberfest submission.
It would be great if I can get some more info to work with

**Comment by Harsh-raj-x07** [comment]
Hey @divyachopra14 can you pls assign me ?

**Comment by manojku1803** [comment]
Hello, Can you please assign this issue to me?

**Comment by ghost** [comment]
assign me this dear,i would like to solve this

**Comment by Abhi-goyal1** [comment]
hello sir ,
please assign me this issue

**Comment by piojagcodes** [comment]
Welcome 
I would like to give a shot. Not mine first hackathon and also last.
Take care
